tests/acceptance/features/apiOcm/share.feature

```diff
@@ -171,3 +266,88 @@ Feature: an user shares resources usin ScienceMesh application
         }
       }
       """
+
+
+  Scenario: users shares file to federation users after accepting invitation
+    Given using server "LOCAL"
```

Comment:
maybw group this with above similar scenario

Reply:
done